nautobot/extras/forms/forms.py

```diff
@@ -584,29 +584,26 @@ class Meta:
         model = ExternalIntegration
         fields = "__all__"
 
-        EXTRA_CONFIG_HELP_TEXT = """
+        HEADERS_HELP_TEXT = """
             Optional user-defined <a href="https://json.org/">JSON</a> data for this integration. Example:
             <pre><code>{
-                "headers": {
-                    "Accept": "application/json",
-                    "Content-Type": "application/json",
-                },
+                "Accept": "application/json",
+                "Content-Type": "application/json"
             }</code></pre>
-            You can render the field value in Jinja2 template as well with the .render_extra_config() helper method.
```

Comment:
Removed mentions of Jinja2 since the `render_*` methods are model methods and the edit UI for the model feels like the wrong place to document these methods. Also, since using the render methods is optional and core doesn't have control over the context variables passed to `render_jinja2`, we can't even provide an example.

Reply:
Makes sense - I see that we document that these APIs exist in the feature documentation, would it be desirable to add an example there? Even something simple like `remote_url = "https://example.com/api/devices?name={{ device.name }}"` with `{"device": <device instance>}` as the template context?